spring-native-image/README.md

```diff
@@ -67,9 +67,12 @@ This project is built using Maven.
 
 ## Run in a Docker Container
 
-As a nice extra, there is a Dockerfile provided with this demo. So, besides building the application JAR, you see a Docker image built at the `mvn clean package` step, pulling the GraalVM container image, `ghcr.io/graalvm/jdk:ol8-java17`, as the JVM.
+you can easily containerise the native executable using the following command:
```

Comment:
As a nice extra, there is a Dockerfile provided with this demo that pulls the GraalVM container image, ghcr.io/graalvm/jdk:ol8-java17, as the JVM. You can easily containerise the native executable using the following command: